packages/headless/src/controllers/quickview/headless-quickview.ts

```diff
@@ -22,6 +22,11 @@ export interface QuickviewOptions {
    * The result to retrieve a quickview for.
    */
   result: Result;
+
+  /**
+   * The maximum file size in bytes rendered by the quickview window.
```

Comment:
I would suggest:

```
The maximum preview size, in bytes, to retrieve. By default, the full preview is retrieved.
```

since this is the part the headless helps with. The place where the content is rendered is up to the developer,

@dlutzCoveo, what do you think?

Reply:
@samisayegh that could work. 
Alternatively, a slightly more simplified phrasing like this could work as well:  `The maximum preview size to retrieve, in bytes. By default, the full preview is retrieved.`